src/tox/Network/Tox.lhs

```diff
@@ -26,10 +26,76 @@ lengths (e.g.  \texttt{4 | 16}), or an inclusive range (e.g. \texttt{[0,
 100]}).  Open ranges are denoted \texttt{[n,]} to mean a minimum length of
 \texttt{n} with no specified maximum length.
 
+\section{TODO: Goals and threat model}
+
+This section should give an idea on what are the goals and non-goals of Tox, so
+that reader
+
+\begin{itemize}
+  \item understands what problems Tox intends to solve
+  \item can validate if they are addresed by this specification
+  \item can make better tradeoffs and decisions in his own reimplementation of the
+    protocol
+\end{itemize}
+
+(TODO: this is just a placeholder; some more technical description should be
+given)
+
+What Tox Does:
+
+\begin{itemize}
+  \item Authentication: user's address is its public key, thus "adding friend"
+    actually means verifying key (false in case of toxme?); drawback is that
```

Comment:
re drawing the line: make clear that basic working of tox is secure (tox id, etc.) but that if you rely on any third party-service, toxme being just an expample - it does not give the same security guarantees. I know, this is not about a website advertizing tox, and that it should be clear for a developer, but better be clear than face criticism for being insecure. All ways to use a service count, and what at the end matters is the user's security. User does not care if part xyz is what he used was secure, when his data leaked it leake, no matter which part leaked it. So anything that is supported should either be save or have a warning about potentially bing unsave. If a provider of any webservice says they hash all passwords, but receive them in plain text via http, then this service cannot claim to be secure. The line there would be that the transfer is part of the service. But if the user writes down his password on a noted next to his computer, this is not part of the service, and this insecurity is caused by the user.
That's how I see it.